packages/ferium-bin/ferium-bin.pacscript

```diff
@@ -0,0 +1,20 @@
+name="ferium-bin"
+pkgname="ferium"
+breaks="${pkgname}"
+replace="${pkgname}-git"
+repology=("project: ${pkgname}")
+version="4.1.5"
+url="https://github.com/gorilla-devs/${pkgname}/releases/download/v${version}/${pkgname}-linux-gnu-nogui.zip"
+maintainer="TwilightBlood <hwengerstickel@pm.me>"
+
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm755 ferium "${STOWDIR}/${name}/usr/bin/${pkgname}"
```

Comment:
Install the shell completions too

Reply:
The binary is the only thing in the archive. Completions are not provided in it.

Reply:
Is it not possible to run the command to generate the completions?

Reply:
Oh I didn't know ferium had the ability to generate completions. Lemme work on that

Reply:
```text
-> ferium complete --help
ferium-complete 4.1.5
Generate shell auto completions to stdout for the specified shell

USAGE:
    ferium complete <SHELL>

ARGS:
    <SHELL>    The shell to generate auto completions for

OPTIONS:
    -h, --help       Print help information
    -V, --version    Print version information
```

Reply:
The AUR PKGBUILD for reference: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ferium

Reply:
https://github.com/pacstall/pacstall-programs/pull/1371